docs/architecture/adr-052-tendermint-mode.md

```diff
@@ -0,0 +1,73 @@
+# ADR 052: Tendermint Mode
+
+## Changelog
+
+* 27-11-2019: Initial draft from ADR-051
+* 13-01-2020: Separate ADR Tendermint Mode from ADR-051
+
+## Context
+
+- Fullnode mode : fullnode mode does not have any capability to participate on consensus
```

Comment:
```suggestion
- Fullnode mode: fullnode mode does not have the capability to become a validator. 
```

I believe participating in consensus is also passing consensus messages around correct?